components/global/Header/Navigation.js

```diff
@@ -27,12 +28,17 @@ export default function Navigation({
 
   useClickEvent(handleClick);
 
+  useEffect(() => {
+    if (isOpenSetter) isOpenSetter(!!active);
```

Comment:
`isOpen` is responsible for telling the `Header` not to hide on scroll if the nav is open.